src/redux/tasks.tsx

```diff
@@ -1,7 +1,7 @@
 import { createSlice } from '@reduxjs/toolkit'
 import type {PayloadAction} from '@reduxjs/toolkit'
 
-interface Task {
+export interface Task {
```

Comment:
Usually if you need to use the same types in different files/ components it is a good practice to create a shared types file. Where you can create all the types and then export them from there